{"Modified":"2020-12-02T17:15:00","Published":"2020-11-30T16:15:00","access":{"authentication":"SINGLE","complexity":"LOW","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[],"cvss":4,"cvss-time":"2020-12-02T17:15:00","cvss-vector":"AV:N/AC:L/Au:S/C:P/I:N/A:N","cwe":"CWE-613","id":"CVE-2020-4696","impact":{"availability":"NONE","confidentiality":"PARTIAL","integrity":"NONE"},"last-modified":"2020-12-02T17:15:00","references":["https://exchange.xforce.ibmcloud.com/vulnerabilities/186789","https://www.ibm.com/support/pages/node/6372528"],"refmap":{"confirm":["https://www.ibm.com/support/pages/node/6372528"],"xf":["ibm-cp4s-cve20204696-session-fixation (186789)"]},"summary":"IBM Cloud Pak for Security 1.3.0.1(CP4S) does not invalidate session after logout which could allow an authenticated user to obtain sensitive information from the previous session. IBM X-Force ID: 186789.","vulnerable_configuration":[{"id":"cpe:2.3:a:ibm:cloud_pak_for_security:1.3.0.1:*:*:*:*:*:*:*","title":"cpe:2.3:a:ibm:cloud_pak_for_security:1.3.0.1:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:ibm:cloud_pak_for_security:1.3.0.1:*:*:*:*:*:*:*"]}